qsiprep/interfaces/anatomical.py

```diff
@@ -314,9 +314,10 @@ class _DesaturateSkullInputSpec(BaseInterfaceInputSpec):
 
 
 class _DesaturateSkullOutputSpec(TraitedSpec):
+    in_file = File(exists=True)
```

Comment:
I don't remember exactly, but I feel like I added these to fix some workflow connection errors that were popping up. I'll try removing them and see what happens, because I don't see them being referenced anywhere. Possible they are remnants of a previous strategy that I forgot to remove after changing tack.